DentOS_Framework/DentOsTestbed/configuration/testbed_config/sit/testbed.json

```diff
@@ -83,6 +83,7 @@
              },
             "serialDev":"/dev/ttyUSB2",
             "baudrate": 115200,
+            "mediaMode":"fiber",
```

Comment:
Maybe it worth to add new keys tov https://github.com/dentproject/testing/tree/main/DentOS_Framework/DentOsTestbed/configuration

Reply:
done